.travis.yml

```diff
@@ -9,16 +9,28 @@ python:
 services:
   - docker
 
+env:
+  - mysql_root_username=root mysql_root_password='12345'
+
+before_install:
+  - sudo apt-get install -y libmysqlclient-dev
+
+addons:
+  apt:
+    update: true
+
+cache:
+  apt: true`
+
 install:
-  - pip install -r test-requirements.txt
-  - docker build -t devopsloft/devopsloft --build-arg REQUIREMENTS=requirements-dev.txt docker/.
+  - pip install -r requirements-dev.txt
```

Comment:
If the application runs in the container, it's redundant to install pip on the host, the tests should run from inside the container, where the application is served

Reply:
Running tests inside the web container will require to install additional packages which might impact on the execution of the tested services.

Reply:
the tests should have zero foot print in the web container